services/getstream/followTopic.js

```diff
@@ -7,6 +7,7 @@ const {
   DEFAULT_TOPIC_PIC_PATH_SIGN,
   DEFAULT_TOPIC_PIC_PATH_ANON
 } = require('../../vendor/getstream/constant');
+const Getstream = require('../../vendor/getstream');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Getstream` module is imported but not used in this hunk. If it's not used elsewhere in the code, consider removing the import to improve readability and maintainability.
```diff
- const Getstream = require('../../vendor/getstream');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->